### ISSUE TITLE
Configure TsLoader options

### ISSUE THREAD

**Issue by ffMathy** [zeit/ncc#486]
Can I configure TsLoader options via this tool? That could be very useful.

**Comment by badeball** [comment]
Indeed. I'd like to specify [`allowTsInNodeModules`](https://github.com/TypeStrong/ts-loader/pull/773), but that seems difficult now.

**Comment by avaer** [comment]
Is there any (possibly hacky) way to do this currently?
